lib/view_model/callbacks.rb

```diff
@@ -181,7 +181,7 @@ def ineligible(view)
     # ARVM synthetic views are considered part of their association and as such
     # are not visited by callbacks. Eligibility exclusion is intended to be
     # library-internal: subclasses should not attempt to extend this.
-    view.is_a?(ViewModel::ActiveRecord) && view.class.synthetic
+    view.is_a?(ViewModel) && view.class.synthetic
```

Comment:
I think this will have interesting side-effects on Replace - it presently expects plain non-Record ViewModels like `SentenceOffsetsView` to not get hit by callbacks and doesn't talk about them in access controls.

Actually, thinking about it now: since `ParentProxyModel` extends `ViewModel` directly, it _already_ didn't get registered or hit by callbacks: why was this change to synthetic necessary?

Reply:
Should be a simple shuffling of the concept up to `ViewModel` with no behavior changes.